esp-link/cgi.c

```diff
@@ -27,10 +27,11 @@ void ICACHE_FLASH_ATTR noCacheHeaders(HttpdConnData *connData, int code) {
   httpdStartResponse(connData, code);
   httpdHeader(connData, "Cache-Control", "no-cache, no-store, must-revalidate");
   httpdHeader(connData, "Pragma", "no-cache");
-  httpdHeader(connData, "Expires", "0");
+  httpdHeader(connData, "Expires", "Mon, 26 Jul 1997 05:00:00 GMT");
```

Comment:
Purely cosmetic and a sign for good programming style. Treating invalid values for "Expires" as **expired** is a side effect, not a prescripted behaviour.